### PR TITLE
Promote CustomCSS and CustomJS to core extensions

### PR DESCRIPTION
**Pull request by Alkarex**
Relates to e.g.
* https://github.com/FreshRSS/FreshRSS/issues/265#issuecomment-983050585

### CODE REVIEW

**Comment by Alkarex** [comment]
We need to make sure their configuration files are saved in a writeable path under `./data/`
See also `./data/extensions-data/` (might be fine for system extensions but not for user extensions, except if we use some kind of prefix)

File: lib/core-extensions/CustomCSS/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# CustomCSS extension
+
+A FreshRSS extension which give ability to create user-specific CSS rules to apply in addition of the actual theme.
+
+To use it, upload this directory in your `./extensions` directory and enable it on the extension panel in FreshRSS. You can add CSS rules by clicking on the manage button.
```
Comment (Frenzie) [inline lib/core-extensions/CustomCSS/README.md:5]: Leftover.

File: lib/core-extensions/CustomJS/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# CustomJS extension
+
+A FreshRSS extension which give ability to create user-specific JS.
+
+To use it, upload this directory in your `./extensions` directory and enable it on the extension panel in FreshRSS. You can add JS by clicking on the manage button.
```
Comment (Frenzie) [inline lib/core-extensions/CustomJS/README.md:5]: Same.

**Comment by triatic** [comment]
Yes please. It will save me having to install custom extensions each time I install freshRSS.

**Comment by Alkarex** [comment]
This was replaced by https://github.com/FreshRSS/FreshRSS/pull/6267

**Pull request closed without merging**